<ui version="4.0" >
 <class>DlgStatusMessageBase</class>
 <widget class="QDialog" name="DlgStatusMessageBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="labelTopLeft" >
       <property name="text" >
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelBottomLeft" >
       <property name="text" >
        <string>Job Number:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QLabel" name="labelTopRight" >
       <property name="text" >
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <item>
        <widget class="QLabel" name="labelBottomRight" >
         <property name="text" >
          <string>Job Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
